FactoryMethod/Projects/Logistics/src/Transport.h

```diff
@@ -0,0 +1,22 @@
+//
+// Created by @PolinaShlepakova on 17.02.2019.
+//
+
+#ifndef LOGISTICS_TRANSPORT_H
+#define LOGISTICS_TRANSPORT_H
+
+#include "Cargo.h"
+#include "Destination.h"
+
+class Transport {
```

Comment:
@PolinaShlepakova,

`Transport` is really good interface!
But both `Ship` and `Truck` have an `_id` field.
Maybe, `Transport` interface could contain `id()` method as `Ship` and `Truck` contain?

Reply:
What exactly do you mean?
ID in Transport instead of derived, its own ID for Transport and for each derived class or ID, implemented in derived, but an interface for it in Transport?

Reply:
I mean 

> ID, implemented in derived, but an interface for it in Transport

Reply:
Ok, will fix that